frontend/test/metabase/scenarios/filters/relative-datetime.cy.spec.js

```diff
@@ -215,23 +215,21 @@ describe("scenarios > question > relative-datetime", () => {
     it("should allow changing values with starting from (metabase#22227)", () => {
       openOrdersTable();
 
-      cy.intercept("POST", "/api/dataset").as("dataset");
       openCreatedAt("Past");
       addStartingFrom();
       setRelativeDatetimeUnit("months");
       setRelativeDatetimeValue(1);
       popover().within(() => {
-        cy.findByText("Add filter").click();
+        cy.button("Add filter").click();
       });
       cy.wait("@dataset");
 
-      cy.intercept("POST", "/api/dataset").as("dataset");
```

Comment:
The same comment as https://github.com/metabase/metabase/pull/23416/files#r900435876